sebs/benchmark.py

```diff
@@ -273,6 +275,28 @@ def copy_code(self, output_dir):
         if os.path.exists(nodejs_package_json):
             shutil.copy2(nodejs_package_json, os.path.join(output_dir, "package.json"))
 
+    #This is for making jar file and add it to docker directory
+    def add_java_output(self, code_dir):
+
+        if self.language_name == "java":
+
+            # Step 1: Move Main.java o src directory
+            src_dir = os.path.join(code_dir, "src", "main", "java")
+            if os.path.exists(code_dir):
+                main_java_path = os.path.join(code_dir, "Main.java")
+                if os.path.exists(main_java_path):
+                    shutil.move(main_java_path, src_dir)
+
+            # Step 2: Run mvn clean install
+            try:
+                # Navigate to the code directory where the pom.xml file is located
+                subprocess.run(['mvn', 'clean', 'install'], cwd=code_dir, check=True, text=True, capture_output=True)
```

Comment:
Typically, we run install steps in a Docker container - see `dockerfiles/{lang}/{platform}`. We create a `build` image for each platform. 

For OpenWhisk, we don't use `build` images because we have `function` images that we deploy. Then why do we run maven here? Shouldn't we run it as a part of building the function image?